js/viz/core/utils.js

```diff
@@ -444,6 +444,50 @@ export function getVizRangeObject(value) {
     }
 }
 
+export function normalizeArcParams(x, y, innerR, outerR, startAngle, endAngle) {
+    let isCircle;
+    let noArc = true;
+    const angleDiff = roundValue(endAngle, 3) - roundValue(startAngle, 3);
```

Comment:
seems like we can get `abs` here to make the following code clearer

Reply:
"roundValue" is also used for exponential numbers, and since this is the output of the old code, I don't want to change it in this place